apps/main/src/modules/reservations/entities/reservation.entity.ts

```diff
@@ -2,16 +2,17 @@ import { AbstractEntity } from '@app/common';
 import { Column, Entity, JoinColumn, ManyToOne, OneToMany } from 'typeorm';
 import { Session } from '../../sessions/entities/session.entity';
 import { ReservationStatus } from '../reservationStatus/entities/reservation-status.entity';
-import { ReservationHallSeats } from './reservationHallSeat.entity';
+import { ReservationHallSeat } from '../reservationHallSeat/entities/reservationHallSeat.entity';
 
 @Entity({
     name: 'reservations'
 })
 export class Reservation extends AbstractEntity<Reservation> {
-    @ManyToOne(() => Session)
+    @ManyToOne(() => Session, { eager: true })
```

Comment:
For example here :D I believe reservation and session entities could be loaded separately for our logic, I mean not always we need to load them together, so I'd like to remove eager:true from here and use join in place, where the relation is required